packages/eslint-config-react-app/index.js

```diff
@@ -213,6 +213,7 @@ module.exports = {
     // https://github.com/gajus/eslint-plugin-flowtype
     'flowtype/define-flow-type': 'warn',
     'flowtype/require-valid-file-annotation': 'warn',
-    'flowtype/use-flow-type': 'warn'
+    'flowtype/use-flow-type': 'warn',
+    'no-restricted-globals': ['warn', 'event']
```

Comment:
yeah, make sense. I missed the fact that in a react app such use case make no sense, as opposite to bad practice in vanilla js.